trade-finance-manager-ui/server/schemas/entra-id-user.schema.entra-id-user-schema.test.ts

```diff
@@ -0,0 +1,9 @@
+import { EntraIdUserSchema } from './entra-id-user.schema';
+import { withEntraIdUserSchemaTests } from './with-entra-id-user-schema.tests';
+
+describe('EntraIdUserSchema', () => {
+  withEntraIdUserSchemaTests({
+    schema: EntraIdUserSchema,
+    getValidObjectWithUpdatedEntraIdUserParams: (entraIdUser) => entraIdUser,
```

Comment:
What is the purpose of `getValidObjectWithUpdatedEntraIdUserParams` please? Seems to just return the same thing?

Reply:
It's to allow for commonisation of `EntraIdUserSchema` tests

Reply:
(sorry @ttbarnes -- might not have seen that we use them in `entra-id-authentication-result-schema` validation as well)